pkg/apis/internal.acorn.io/v1/service.go

```diff
@@ -29,6 +29,7 @@ type ServiceInstance struct {
 type ServiceInstanceStatus struct {
 	Conditions []Condition `json:"conditions,omitempty"`
 	Endpoints  []Endpoint  `json:"endpoints,omitempty"`
+	HasService bool        `json:"hasService,omitempty"`
```

Comment:
nit: it's not conventional here, but if it's not too burdensome, it would be nice to start adding descriptive godocs comments for net-new stuff

I'm assuming this propagates to the CRD's openapi spec and k8s API discovery as well (though I'm not sure if wrangler handle's that in it's CRD generation)